examples/custom/class/README.md

```diff
@@ -0,0 +1,8 @@
+Sometimes is it convenient to add custom HTML classes to tags generated by Bokeh (i.e. for creating a custom CSS for a single element only).
+
+Until proper support will be added (#5471) it is possible to properly extend Bokeh element View by adding a custom class.
+
+Current example demonstrates how to added custom HTML classes to derived Row/Column layouts.
+
```

Comment:
"class" seems like a very undescriptive name for the README.md

The README's should include a how to run and probably shouldn't include references to future fixes or personal touches like the signature here.